docs/Team_Agreements.md

```diff
@@ -13,16 +13,33 @@
 - Add the Epic
 - Add the Sprint, if needed
 
+### Grooming cards
```

Comment:
```suggestion
### Refining cards
```
Due to some connotations of the word "grooming", "backlog refinement" is now preferred